### ISSUE TITLE
2.19.3 fails with Error: fatal: pathspec 'undefined' did not match any files

### ISSUE THREAD

**Issue by kurtinatlanta** [jscutlery/semver#449]
I am using npm 8.5 and the --legacy-peer-deps switch when running npm install.

My environment: @nrwl/* 13.8.3, @jscutlery/semver 2.19.3.

This is the version entry in project.json:
```
    "version": {
      "executor": "@jscutlery/semver:version",
      "options": {
        "commitMessageFormat": "chore(${projectName}): release version ${version}",
        "skipProjectChangelog": true
      }
    },
```
Here is the stack trace (with sanitized root paths):
```
Error: fatal: pathspec 'undefined' did not match any files

    at ./node_modules/@jscutlery/semver/src/executors/version/utils/git.js:62:186
    at Observable.init [as _subscribe] (./node_modules/@jscutlery/semver/node_modules/rxjs/dist/cjs/internal/observable/throwError.js:8:64)
    at Observable._trySubscribe (./node_modules/@jscutlery/semver/node_modules/rxjs/dist/cjs/internal/Observable.js:41:25)
    at ./node_modules/@jscutlery/semver/node_modules/rxjs/dist/cjs/internal/Observable.js:35:31
    at Object.errorContext (./node_modules/@jscutlery/semver/node_modules/rxjs/dist/cjs/internal/util/errorContext.js:22:9)
    at Observable.subscribe (./node_modules/@jscutlery/semver/node_modules/rxjs/dist/cjs/internal/Observable.js:26:24)
    at ./node_modules/@jscutlery/semver/node_modules/rxjs/dist/cjs/internal/operators/catchError.js:17:31
    at OperatorSubscriber._this._error (./node_modules/@jscutlery/semver/node_modules/rxjs/dist/cjs/internal/operators/OperatorSubscriber.js:43:21)
    at OperatorSubscriber.Subscriber.error (./node_modules/@jscutlery/semver/node_modules/rxjs/dist/cjs/internal/Subscriber.js:60:18)
    at OperatorSubscriber.Subscriber._error (./node_modules/@jscutlery/semver/node_modules/rxjs/dist/cjs/internal/Subscriber.js:84:30)
```
I'm going to rollback to the previous release and see if that helps. What else can I look for that would help?

Rolling back to 2.18.2 makes the issue go away, so there must be a breakage in 2.19.*

**Comment by edbzn** [comment]
@kurtinatlanta Thanks for the report, this has been fixed in 2.19.4.

**Comment by kurtinatlanta** [comment]
Much better. Thank you very much!
